install/kubernetes/helm/istio/values.yaml

```diff
@@ -86,6 +86,40 @@ ingress:
       enabled: false
       port: 32000
 
+#
+# ingress-gateway configuration
+#
+ingress-gateway:
+  enabled: true
+  serviceAccountName: default
+  imagePullPolicy: IfNotPresent
+  autoscaleMin: 1
+  autoscaleMax: 1
+  resources: {}
+# limits:
+#  cpu: 100m
+#  memory: 128Mi
+# requests:
+#  cpu: 100m
+#  memory: 128Mi
+
+#
+# egress-gateway configuration
+#
+egress-gateway:
```

Comment:
same comments apply to egress-gateway as ingress-gateway

Reply:
we had to change this to sidecarInjector in values.yaml.  it is probably good to change to that format to be consistent too.